src/simtools/testing/validate_output.py

```diff
@@ -64,9 +64,20 @@ def _validate_output_files(config, integration_test):
 
 def _test_simtel_cfg_files(config, integration_test, from_command_line, from_config_file):
     """Test simtel cfg files."""
-    test_simtel_cfg_file = integration_test.get("TEST_SIMTEL_CFG_FILES", {}).get(
-        from_command_line or from_config_file
-    )
+    test_simtel_cfg_file = None
+    test_simtel_cfg_files = integration_test.get("TEST_SIMTEL_CFG_FILES", {})
+    if isinstance(from_command_line, list):
+        for version in from_command_line:
+            if version in test_simtel_cfg_files:
+                test_simtel_cfg_file = test_simtel_cfg_files[version]
+                break
+    elif isinstance(from_config_file, list):
+        for version in from_config_file:
+            if version in test_simtel_cfg_files:
+                test_simtel_cfg_file = test_simtel_cfg_files[version]
+                break
+    else:
+        test_simtel_cfg_file = test_simtel_cfg_files.get(from_command_line or from_config_file)
```

Comment:
This is really ugly code, but it's more or less the only way to protect against the case when one of the model versions is a list. A simpler option is to simply check first if `test_simtel_cfg_files` is not empty and if not, continue with simpler old code, otherwise return. That is,

```
test_simtel_cfg_files = integration_test.get("TEST_SIMTEL_CFG_FILES", {})
if not test_simtel_cfg_files:
    return

test_simtel_cfg_file = integration_test.get("TEST_SIMTEL_CFG_FILES", {}).get(
        from_command_line or from_config_file
    )
...
```

Currently we do not have a case of multiple versions and `TEST_SIMTEL_CFG_FILES`, so nothing will break, but it will if we ever add a test like that.

Reply:
Would this be better? Less repetitive.

```
def _test_simtel_cfg_files(config, integration_test, from_command_line, from_config_file):
    """Test simtel cfg files."""
    cfg_files = integration_test.get("TEST_SIMTEL_CFG_FILES", {})
    sources = from_command_line if isinstance(from_command_line, list) else (
              from_config_file if isinstance(from_config_file, list) else [from_command_line or from_config_file])
    
    for version in sources:
        cfg = cfg_files.get(version)
        if cfg:
            _validate_simtel_cfg_files(config, cfg)
            break
```

Reply:
Yes, this is shorter and less repetitive, so I will change to it. However, the original question was whether the concept of protecting against the lists even warrants the extra complication at this point. 
I take it from your suggestion that you think so and we should keep it.

Reply:
Yes, keep it.